core/domain/exp_jobs_one_off_test.py

```diff
@@ -92,6 +92,207 @@ def run_job_for_deleted_exp(
+                param_changes=[],
+                skin_customizations={},
+            )
+            rights_manager.create_new_exploration_rights('exp_id3', 'uid_3')
+            commit_message = 'New exploration created with title \'title\'.'
+            exp_model3.commit('uid_3', commit_message, [{
+                'cmd': 'create_new',
+                'title': 'title',
+                'category': 'category',
+            }])
+
+            exp_model1 = (
+                exp_models.ExplorationModel.get_by_id('exp_id1'))
+            exp_model2 = (
+                exp_models.ExplorationModel.get_by_id('exp_id2'))
+            exp_model3 = (
+                exp_models.ExplorationModel.get_by_id('exp_id3'))
+            self.assertIsNotNone(exp_model1.skill_tags)
+            self.assertIsNotNone(exp_model2.default_skin)
+            self.assertIsNotNone(exp_model3.skin_customizations)
+            self.assertIn('skill_tags', exp_model1._values) # pylint: disable=protected-access
+            self.assertIn('skill_tags', exp_model1._properties) # pylint: disable=protected-access
+            self.assertIn('default_skin', exp_model2._values) # pylint: disable=protected-access
+            self.assertIn('default_skin', exp_model2._properties) # pylint: disable=protected-access
+            self.assertIn('skin_customizations', exp_model3._values) # pylint: disable=protected-access
+            self.assertIn('skin_customizations', exp_model3._properties) # pylint: disable=protected-access
+
+            output = self._run_one_off_job()
+            self.assertItemsEqual(
+                [['SUCCESS_REMOVED - ExplorationModel', 3]], output)
+
+            migrated_exp_model1 = (
+                exp_models.ExplorationModel.get_by_id('exp_id1'))
+            migrated_exp_model2 = (
+                exp_models.ExplorationModel.get_by_id('exp_id2'))
+            migrated_exp_model3 = (
+                exp_models.ExplorationModel.get_by_id('exp_id3'))
+            self.assertNotIn(
+                'skill_tags', migrated_exp_model1._values)  # pylint: disable=protected-access
+            self.assertNotIn(
+                'skill_tags', migrated_exp_model1._properties)  # pylint: disable=protected-access
+            self.assertNotIn(
+                'default_skin', migrated_exp_model2._values)  # pylint: disable=protected-access
+            self.assertNotIn(
+                'default_skin', migrated_exp_model2._properties)  # pylint: disable=protected-access
+            self.assertNotIn(
+                'skin_customizations', migrated_exp_model3._values)  # pylint: disable=protected-access
+            self.assertNotIn(
+                'skin_customizations', migrated_exp_model3._properties)  # pylint: disable=protected-access
+
+            # Run job twice.
+            output = self._run_one_off_job()
+            self.assertItemsEqual(
+                [['SUCCESS_ALREADY_REMOVED - ExplorationModel', 3]], output)
```

Comment:
Please separate it into own test, we try to have tests written so that they test only one thing. This applies to more things in this test.

Reply:
I wanted to ask one thing, what does the number 3 here refers to?

Reply:
Done